evap/development/management/commands/run.py

```diff
@@ -9,6 +9,8 @@ class Command(BaseCommand):
     help = 'Execute "runserver 0.0.0.0:8000"'
 
     def handle(self, *args, **options):
+        self.stdout.write('Executing "manage.py scss"')
+        execute_from_command_line(['manage.py', 'scss'])
```

Comment:
how fast is that?

Reply:
On my machine™ around 1s normally, but I experienced a runtime of 2s. I think this is fine for `scss`, but I need to investigate something for the follow-up PR as typescript takes around 7s which is more noticable. This is just convenience so that the workflow does not change “just run `management.py run` and you are all set”.
I’m tending to run those in parallel (rather unlikely that the first browser request is made before sass has finished), but I don’t know how I would produce some sensible output.